src/main/java/org/ihtsdo/snowowl/authoring/single/api/rest/UiStateController.java

```diff
@@ -17,85 +17,87 @@
 
 @Api("UI State")
 @RestController
-@RequestMapping(produces={MediaType.APPLICATION_JSON_VALUE})
+@RequestMapping(produces = MediaType.APPLICATION_JSON_VALUE)
 public class UiStateController {
```

Comment:
All the changes in this talk look like refactoring?

Reply:
Yes that's right, I've adjusted the parameter styling on the endpoint methods to follow your guidelines.